services/alerts/src/AlertsProvider.tsx

```diff
@@ -6,7 +6,7 @@ import { AlertsManagerAlert, AlertsManager } from './types'
 
 export const AlertsProvider = ({ children }: { children: React.ReactNode }) => {
     const [alerts, setAlerts] = useState<AlertsManagerAlert[]>([])
-    const [alertsManager] = useState<AlertsManager>(() =>
+    const [alertsManager] = useState<AlertsManager>(
```

Comment:
I think this should keep the `() => makeAlertsManager(setAlerts)` form, otherwise `makeAlertsManager` is called every render